### PR TITLE
Add release notes to arcade issue template

### PR DESCRIPTION
**Pull request by jonfortescue**
Closes https://github.com/dotnet/arcade/issues/11432.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/default-arcade-issue-template.md [hunk 1]
```diff
@@ -12,3 +12,6 @@ assignees: ''
 - [ ] This issue is causing unreasonable pain
 
 <!-- Write your issue description below. -->
+
+<!-- For internal use only; put release notes here -->
```
Comment (MattGal) [inline .github/ISSUE_TEMPLATE/default-arcade-issue-template.md:16]: feels like this should include a link to instructions, since folks could do things like be too wordy or provide sensitive internal information on the public GH issue.

**Review by MattGal — APPROVED** [review]

**Review by garath — DISMISSED** [review]

**Review by MattGal — APPROVED** [review]
